### ISSUE TITLE
Multiple SQL injections

### ISSUE THREAD

**Issue by tch1bo** [HarshShah1997/Shopping-Cart#1]
Hello,

I noticed that some of the SQL queries are not sanitized properly, which opens a possibility of SQL Injection attacks (https://www.owasp.org/index.php/SQL_Injection).

As a Proof of Concept consider: http://localhost:5000/productDescription?productId=1%20or%201==1%20--+

Vulnerable lines in main.py are: 20, 91, 107, 128, 144, 213, 226, 246, 263.

To fix those, i would suggest using prepared statements, as, for example, in line 148.

**Comment by HarshShah1997** [comment]
Hi,

Thanks for mentioning the vulnerability. This project was something I created to learn web development and I have stopped working on it since 3 years. As such, there are many security vulnerabilities and poorly optimized present in the code. I do not plan to do any further development in this project but I am happy to accept any pull request. My suggestion for you is to send a pull request containing the fix for SQL Injection vulnerability and I will merge it into the master.
